Client/src/app/_services/account.service.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ReplaySubject } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { User } from '../_models/user';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AccountService {
+  private accountUrl = environment.apiUrl + 'account';
+  private currentUser = new ReplaySubject<User>(1);
+  currentUser$ = this.currentUser.asObservable();
```

Comment:
Can you explain how this works and what it means ?

Reply:
https://github.com/Code2Gether-Discord/JokesOnYou/wiki/Account-Service-in-Angular-Client